package.json

```diff
@@ -11,7 +11,7 @@
     "e2e": "node ./e2e/generate-fixtures-json.js && jest --config=e2e/jest-e2e.config.js",
     "postinstall": "lerna bootstrap",
     "test:watch": "jest --watch",
-    "version": "json -I -f assets/manifest.json -e \"this.version='`json -f package.json version`'\" && git add assets/manifest.json && json -I -f packages/core/package.json -e \"this.version='`json -f package.json version`'\" && git add packages/core/package.json",
+    "version": "dot-json assets/manifest.json version $npm_package_version && dot-json packages/core/package.json version $npm_package_version && git add assets/manifest.json packages/core/package.json",
     "patch-permissions": "json -I -f dist/manifest.json -e 'this.permissions.push(\"https://api.github.com/\")'",
```

Comment:
Why do you add this permission at build time?

Reply:
There was an issue with Firefox not being able to make request against the GitHub API without having `https://api.github.com` explicit being set

Reply:
You can include it for chrome as well

Reply:
but wouldn't this prompt a dialog asking for additional permissions when the update arrives?

Reply:
I don’t remember, but since it’s GitHub itself it’s not a big deal

Reply:
Yeah, maybe I'm overthinking this. Alos it can be solved by communicating why this is needed now. Let's do it. Do you mind upgrading the manifest.json file as part of this PR?